src/main/java/com/EstimatedDeliveryDate/Entity/ShipDetail.java

```diff
@@ -0,0 +1,15 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+/*Java object to extract details from request body*/
+@Getter
+@Setter
+@NoArgsConstructor
+public class ShipDetail {
+	private String shipDate;
+	private String shipFrom;
+	private String shipTo;
```

Comment:
Add better name for the variable so that it is more readable. Change to shipFromPostalCode.